integration/routes/metadata.test.ts

```diff
@@ -0,0 +1,31 @@
+import client from '../client';
+
+describe('POST /metadata', () => {
+
+  const metadata = {
+    "firstName": "Aidan",
+    "dob": "1999-09-09"
+  }
+
+  const expectedSaveResponse = {documentId: expect.any(String), url: expect.any(String), fields: expect.objectContaining({success_action_status: "201",
```

Comment:
nitpick: this line is very long - not sure why eslint didn't pick it up but maybe could do with some prettyfying?

Reply:
maybe time to "upgrade" your editor Bogdan? :-P

Reply:
is there a better one? :O